dateutil/parser/_parser.py

```diff
@@ -57,7 +57,7 @@
 # take off their plate.
 class _timelex(object):
     # Fractional seconds are sometimes split by a comma
-    _split_decimal = re.compile("([.,])")
+    _split_decimal = re.compile("([.])")
```

Comment:
If we're moving the logic into the parser, we don't have to use a compiled regular expression, since I believe `token.split('.')` will work.

Reply:
Good idea.

IIRC the status of this is that it needs to entail [zero behavioral changes](https://github.com/dateutil/dateutil/pull/732#issuecomment-401887004).  That is not yet the case with this PR; I'll move this up the priority list.